src/cobalt.rs

```diff
@@ -177,6 +232,7 @@ pub fn build(config: &Config) -> Result<()> {
         }
 
         let mut context = post.get_render_context(&simple_posts_data);
+        context.set_val("data", Value::Object(data_map.clone()));
```

Comment:
So this puts `data` in the root object.  What about following what hugo and jekyll do and call it `site.data`?

Reply:
Fine with me. Was actually surprised there was no `site`-object yet (or at least I couldn't find it), but didn't want to add it, because this comes with a bunch of other things attached to it(on [jekyll at least](https://jekyllrb.com/docs/variables/#site-variables) and at least `site.time` I am using ;) )), unrelated to this feature.

Does this `site` already exist? Otherwise we might want to open another compatibility-issue for it?

Reply:
`site` does not already exist.  I was considering adding it but when you mentioned working on this, I decided to wait to not run into merge conflicts.